### PR TITLE
fixes 3326

### PR DESCRIPTION
**Pull request by cdhanna**
fixes 3326 by not throwing exception if source doesn't exist.

### CODE REVIEW

**Review by gabrielbeamable — APPROVED** [review]

File: microservice/microserviceTests/CronBuilderTests/JobConversionTests.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+	[TestCase]
+	public void CanDeserializeJobWithoutSource()
+	{
+		var job = new JobDefinition
+		{
+			id = "test",
+			name = "test",
+			owner = "test",
+			source = new OptionalString(),
```
Comment (allister-beamable) [inline microservice/microserviceTests/CronBuilderTests/JobConversionTests.cs:18]: The semantics here are that `new OptionalString()` with no constructor argument is an empty optional string? That is, `new OptionalString()` is the equivalent of Scala`s `None`?

File: client/Packages/com.beamable.server/CHANGELOG.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [1.19.18]
+### Fixed
+- `BeamScheduler` can deserialize jobs without a `source` field.
+- `AssumeNewUser` does not allow `userId` that is not a positive value
```
Comment (allister-beamable) [inline client/Packages/com.beamable.server/CHANGELOG.md:11]: This seems not-relevant to the current changes.

**Review by allister-beamable — CHANGES_REQUESTED** [review]
Code looks good, but the changelog appears to have a rogue entry about AssumeNewUser.

**Review by allister-beamable — APPROVED** [review]
I was wrong about the rogue changelog entry. The AssumeNewUser line was already there, it just moved along with the `### Fixed` header.

File: microservice/microserviceTests/CronBuilderTests/JobConversionTests.cs (hunk 1 above)
Comment (cdhanna) [reply microservice/microserviceTests/CronBuilderTests/JobConversionTests.cs:18]: correct- that is a `None`

**Pull request merged**